benchmarks/throughput.dart

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2021, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:math';
+
+import 'package:cronet/cronet.dart';
+
+class CronetThroughputBenchmark {
```

Comment:
ThroughputBenchmark with subclasses CronetThroughputBenchmark and DartIOThroughputBenchmark.